card/src/main/res/values-bg-rBG/strings.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ Copyright (c) 2019 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by ran on 13/3/2019.
+  -->
+
+<resources>
+    <string name="checkout_card_number_hint">Номер на карта</string>
+    <string name="checkout_card_holder_hint">Име на картата</string>
+    <string name="checkout_card_expiry_date_hint">Дата на валидност</string>
+    <string name="checkout_card_expiry_date_optional_hint">Дата на валидност (незадължително)</string>
+    <string name="checkout_card_security_code_hint">Код за сигурност</string>
+    <string name="checkout_card_security_code_optional_hint">Код за сигурност (незадължително)</string>
+    <string name="checkout_kcp_birth_date_or_tax_number_hint">Дата на раждане на картодържателя (ГГГГММДД) или корпоративен регистрационен номер (10 цифри)</string>
+    <string name="checkout_kcp_tax_number_hint">Корпоративен регистрационен номер (10 цифри)</string>
+    <string name="checkout_kcp_password_hint">Първите 2 цифри от паролата на картата</string>
+
+    <string name="checkout_card_number_not_valid">Невалиден номер на карта</string>
+    <string name="checkout_expiry_date_not_valid">Невалидна дата на валидност</string>
+    <string name="checkout_expiry_date_not_valid_too_old">Въведете валидна дата на изтичане</string>
+    <string name="checkout_expiry_date_not_valid_too_far_in_future">Въведете валидна дата на изтичане</string>
+    <string name="checkout_security_code_not_valid">Невалиден CVC/CVV формат</string>
+    <string name="checkout_holder_name_not_valid">Името на притежателя е невалидно</string>
+    <string name="checkout_social_security_number_not_valid">Невалидно въвеждане</string>
+    <string name="checkout_kcp_birth_date_or_tax_number_invalid">Невалидна рождена дата или регистрационен номер на картодържателя</string>
+    <string name="checkout_kcp_password_invalid">Невалидна парола</string>
+    <string name="checkout_card_postal_not_valid">Входният формат не е валиден.</string>
+    <string name="checkout_card_brand_not_supported">Въведената марка на картата не се поддържа</string>
+    <string name="checkout_card_formatted_brand_name_not_supported">%s не се поддържа</string>
+
+    <string name="checkout_card_installments_title">Брой вноски</string>
+    <string name="checkout_card_installments_option_regular">%s месеци</string>
+    <string name="checkout_card_installments_option_regular_with_price">%sx %s</string>
```

Comment:
This key is "special", we need to go back to the original translations and verify which `%s` should be `%1$s` and which one should be `%2$s` and manually change them :/